cmd/edgectl/aes_install.go

```diff
@@ -0,0 +1,290 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/datawire/ambassador/pkg/k8s"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func aesInstallCmd() *cobra.Command {
+	res := &cobra.Command{
+		Use:   "install",
+		Short: "Install the Ambassador Edge Stack in your cluster",
+		Args:  cobra.ExactArgs(0),
+		RunE:  aesInstall,
+	}
+	_ = res.Flags().StringP(
+		"context", "c", "",
+		"The Kubernetes context to use. Defaults to the current kubectl context.",
+	)
+	_ = res.Flags().StringP(
+		"namespace", "n", "",
+		"The Kubernetes namespace to use. Defaults to kubectl's default for the context.",
+	)
+
+	return res
+}
+
+func aesInstall(cmd *cobra.Command, args []string) error {
+	metrics := NewMetrics()
+	_ = metrics.Report("install")
+
+	// Display version information
+	fmt.Printf("-> Installing the Ambassador Edge Stack %s\n", Version)
```

Comment:
Yes, that's correct. We could also insist that you have the latest Edge Control. That would allow us to make the installation process more complicated in the future, assuming this install method gains traction. Anyhow, yes, there should be a FIXME or TODO here.